projects/addon-doc/components/code/index.html

```diff
@@ -15,6 +15,7 @@
             appearance="outline-grayscale"
             size="s"
             class="t-copy-button"
+            title="Copy"
```

Comment:
Should we use the following token ?

https://github.com/taiga-family/taiga-ui/blob/24572b1cdc7e0bbd48b3bcbf7faf3960e80c4372/projects/kit/tokens/i18n.ts#L41

Reply:
Also we can put it in content of tuiIconButton instead of title that shows on hover.

BTW, what's up with this part, a few lines above, why isn't it formatted properly?
```html
        <button type="button"
            tuiIconButton
```